widget/embedded/src/pages/Home.tsx

```diff
@@ -116,6 +118,10 @@ export function Home() {
     meta: { tokens, blockchains },
     loadingStatus: loadingMetaStatus,
   } = useMetaStore();
+  const referrer = new URLSearchParams(location.search).get(
+    SearchParams.REFERRER
+  );
+  useReferrer({ referrer, loadingStatus: loadingMetaStatus });
```

Comment:
Auto connect doesn't trigger on other pages . you should move it to widget root.

Reply:
fixed